.storybook/stories/SelectableTableBatchActions/ModalConfirmDelete.tsx

```diff
@@ -2,23 +2,29 @@ import { useMemo, useState } from 'react';
 import { ButtonGroup, Button } from '@ag.ds-next/react/button';
 import { Text } from '@ag.ds-next/react/text';
 import { Modal } from '@ag.ds-next/react/modal';
+import { getRequiredCloseHandler } from '@ag.ds-next/react/src/getCloseHandler';
 import { plural, RowData } from './utils';
 
 export type ModalConfirmDeleteProps = {
 	isOpen: boolean;
 	onConfirm: () => void;
-	onDismiss: () => void;
+	/** @deprecated use `onClose` instead */
```

Comment:
This is just a story as well, so we can remove this

Reply:
removed